Library/Homebrew/cmd/install-vendor.sh

```diff
@@ -0,0 +1,117 @@
+source "$HOMEBREW_LIBRARY/Homebrew/utils/lock.sh"
+
+if [[ -n "$HOMEBREW_OSX" ]]
+then
+  if [[ "$HOMEBREW_PROCESSOR" = "Intel" ]]
+  then
+    git_URL=""
+    git_SHA=""
+    ruby_URL="https://github.com/xu-cheng/homebrew-portable/releases/download/20160512/portable-ruby-2.0.0-p648.x86_64-darwin9.tar.gz"
```

Comment:
I have updated to use 32bit ruby. 

Also FYI, if we move to ruby 2.1, I will be able to build universal version ruby.